phpBB/includes/acp/acp_storage.php

```diff
@@ -51,24 +63,31 @@ class acp_storage
 	/** @var string */
 	public $u_action;
 
+	/** @var mixed */
+	protected $state;
+
 	/**
-	 *  @param string $id
+	 * @param string $id
 	 * @param string $mode
 	 */
 	public function main($id, $mode)
 	{
 		global $phpbb_container, $phpbb_dispatcher;
 
 		$this->config = $phpbb_container->get('config');
-		$this->lang = $phpbb_container->get('language');
```

Comment:
actually it is better to use the language service than the user object :)

Reply:
I remember i did this for a reason, but i don't remember, maybe because of the strings that come from the extensions

Reply:
user->lang() does nothing else than calling lang->lang()